cf-monitord/env_monitor.c

```diff
@@ -269,7 +269,7 @@ void MonitorStartServer(EvalContext *ctx, const Policy *policy)
         Bundle *bp = PolicyAppendBundle(monitor_cfengine_policy, NamespaceDefault(), "monitor_cfengine_bundle", "agent", NULL, NULL);
         PromiseType *tp = BundleAppendPromiseType(bp, "monitor_cfengine");
 
-        pp = PromiseTypeAppendPromise(tp, "the monitor daemon", (Rval) { NULL, RVAL_TYPE_NOPROMISEE }, NULL, NULL);
+        pp = PromiseTypeAppendPromise(tp, "the monitor daemon", RvalNULL(), NULL, NULL);
```

Comment:
I had a prep commit but rebasing it was really painful and I squashed it. I can split it out.